src/Apps/Artwork/Components/RelatedWorks/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTracking } from "react-tracking"
+
+import { ContextModule } from "@artsy/cohesion"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import { Box, Skeleton, Spacer } from "@artsy/palette"
+
+import { useSystemContext } from "System"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import {
+  Header,
+  HeaderPlaceholder,
+} from "Apps/Artwork/Components/OtherWorks/Header"
+import ArtworkGrid, { ArtworkGridPlaceholder } from "Components/ArtworkGrid"
+import { RelatedWorksQuery } from "__generated__/RelatedWorksQuery.graphql"
+import { RelatedWorks_artwork$data } from "__generated__/RelatedWorks_artwork.graphql"
+
+interface RelatedWorksProps {
+  artwork: RelatedWorks_artwork$data
+}
+
+export const RelatedWorks: React.FC<RelatedWorksProps> = ({ artwork }) => {
+  const { trackEvent } = useTracking()
+
+  const artworksConnection = artwork.layer?.artworksConnection
+
+  if (!artworksConnection?.edges?.length) {
+    return null
+  }
+
+  return (
+    <Box data-testid={ContextModule.relatedWorksRail}>
+      <Header title="Related works" />
+
+      <Spacer mt={4} />
+
+      <ArtworkGrid
+        contextModule={ContextModule.relatedWorksRail}
+        artworks={artworksConnection}
+        columnCount={[2, 3, 4, 4]}
+        onBrickClick={() =>
+          trackEvent({
+            type: DeprecatedSchema.Type.ArtworkBrick,
+            action_type: DeprecatedSchema.ActionType.Click,
+            context_module: DeprecatedSchema.ContextModule.RelatedWorks,
+          })
+        }
```

Comment:
Confirmed that this event flows through to the Segment debugger with the same shape as the old one:

```
analytics.track('Click', {
  context_module: 'RelatedWorks',
  type: 'Artwork brick'
});
```